proto/regen/ecocredit/v1/query.proto

```diff
@@ -644,3 +696,61 @@ message BatchBalanceInfo {
   // cancelled, updated with a reduced quantity, or processed.
   string escrowed_amount = 5;
 }
+
+// ClassCreatorInfo is the human-readable class creator information.
+message ClassCreatorInfo {
+  // address is the address that is allowed to create credit classes
+  string address = 1;
+}
+
+// QueryAllowedClassCreatorsRequest is the Query/AllowedClassCreators request
+// type.
+//
+// Since Revision 1
+message QueryAllowedClassCreatorsRequest {
+  // pagination defines an optional pagination for the request.
+  cosmos.base.query.v1beta1.PageRequest pagination = 1;
+}
+
+// QueryAllowedClassCreatorsResponse is the Query/AllowedClassCreators response
+// type.
+//
+// Since Revision 1
+message QueryAllowedClassCreatorsResponse {
+  // class_creators is the list of allowed credit class creators.
+  repeated ClassCreatorInfo class_creators = 1;
+
+  // pagination defines the pagination in the response.
+  cosmos.base.query.v1beta1.PageResponse pagination = 2;
+}
+
+// QueryCreditClassAllowlistEnabledRequest is the
+// Query/CreditClassAllowlistEnabled request type.
+//
+// Since Revision 1
+message QueryCreditClassAllowlistEnabledRequest {}
+
+// QueryCreditClassAllowlistEnabledResponse is the
+// Query/CreditClassAllowlistEnabled response type.
+//
+// Since Revision 1
+message QueryCreditClassAllowlistEnabledResponse {
+  // allowlist_enabled is the state/AllowListEnabled type.
+  AllowListEnabled allowlist_enabled = 1;
+}
```

Comment:
I don't think we need to use the state message. We should should use `bool allowlist_enabled`.